src/babel/index.js

```diff
@@ -172,9 +172,20 @@ function jsxAttr(t, name, value) {
   return t.jSXAttribute(t.jSXIdentifier(name), t.jSXExpressionContainer(value))
 }
 
+const identifierRegex = /^[a-zA-Z][a-zA-Z0-9]*$/
+function objectKey(t, key) {
```

Comment:
This seems to follow the logic from https://babeljs.io/docs/en/babel-types#objectproperty with the exception of expressions, which we don't support